book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,34 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
+
+### OVERVIEW
+This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration."
+### PLUGINS PREPERATIONS
+
+#### Add Single Bundle Plugin to project configuration
+Add single bundle plugin and select versions of react-native, react and react-native-zapp-bridge you want to use. Save plugin configuration and rebuild. Please note rebuild is necesery every time you change versions or adding/removing other react native plugins.
+
+![Single Bundle Configuration]( https://assets-production.applicaster.com/applicaster-employees/zapp_team/anna_bauza/react_native/single-bundle-config.png  "Single Bundle Configuration")
+
+#### Make sure plugins support selected versions
+Your React Native plugins need to be build and published to NPM for selected versions of react and react-native.
+
+
+#### Migrate your plugins
```

Comment:
might consider adding this in the general android guide - or at least a line about it - so people building new plugins would make them migratable or be aware of what needs to be done from the get go

Reply:
great point Liviu, thanks!